Sources/ConfidenceProvider/ConfidenceFeatureProvider.swift

```diff
@@ -86,7 +86,7 @@ public class ConfidenceFeatureProvider: FeatureProvider {
     }
 
     private func updateConfidenceContext(context: EvaluationContext, removedKeys: [String] = []) {
-        confidence.putContext(context: ConfidenceTypeMapper.from(ctx: context), removeKeys: removedKeys)
+        confidence.putContextLocal(context: ConfidenceTypeMapper.from(ctx: context), removeKeys: removedKeys)
```

Comment:
Important bug fix in the OF layer: fetching and activation via dedicated API calls, and this API call should only modify the context to be set in-memory for the session

Reply:
Good catch! I pushed a fix where `putContextLocal` is only called from `initialize`